<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>KrigingSettingsWidget</class>
 <widget class="QWidget" name="KrigingSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>378</width>
    <height>214</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Variogram model</string>
     </property>
     <property name="buddy">
      <cstring>comboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBox">
     <item>
      <property name="text">
       <string>Exponential</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Gaussian</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Spherical</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Variogram sill</string>
     </property>
     <property name="buddy">
      <cstring>lineSill</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineSill"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Variogram radius A</string>
     </property>
     <property name="buddy">
      <cstring>lineRadiusA</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineRadiusA"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Variogram radius B</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="lineRadiusB"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Rotation Angle</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="lineRotationAngle"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Variogram nugget</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineNugget"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
